quality-estimation/modelfront.md

```diff
@@ -0,0 +1,757 @@
+---
+autogenerated: true
+nav_order: 853
+layout: quality_estimation
+title: ModelFront
+description: Translation quality prediction
+id: modelfront
+company: Modelfront
+parent: Quality estimation
+urls:
+- https://modelfront.com
+- https://docs.modelfront.com/api/
+supported_languages:
+- slug: afrikaans
+  code: af
+  base_code: af
+  name: Afrikaans
+  variant_name: null
+- slug: amharic
+  code: am
+  base_code: am
+  name: Amharic
+  variant_name: null
+- slug: arabic
+  code: ar
+  base_code: ar
+  name: Arabic
+  variant_name: null
+- slug: assamese
+  code: as
+  base_code: as
+  name: Assamese
+  variant_name: null
+- slug: azerbaijani
+  code: az
+  base_code: az
+  name: Azerbaijani
+  variant_name: null
+- slug: bashkir
+  code: ba
+  base_code: ba
+  name: Bashkir
+  variant_name: null
+- slug: belarusian
+  code: be
+  base_code: be
+  name: Belarusian
+  variant_name: null
+- slug: bulgarian
+  code: bg
+  base_code: bg
+  name: Bulgarian
+  variant_name: null
+- slug: bengali
+  code: bn
+  base_code: bn
+  name: Bengali
+  variant_name: null
+- slug: tibetan
+  code: bo
+  base_code: bo
+  name: Tibetan
+  variant_name: null
+- slug: bosnian
+  code: bs
+  base_code: bs
+  name: Bosnian
+  variant_name: null
+- slug: catalan
+  code: ca
+  base_code: ca
+  name: Catalan
+  variant_name: null
+- slug: cebuano
+  code: ceb
+  base_code: ceb
+  name: Cebuano
+  variant_name: null
+- slug: sorani-kurdish
+  code: ckb
+  base_code: ckb
+  name: Sorani Kurdish
+  variant_name: null
+- slug: corsican
+  code: co
+  base_code: co
+  name: Corsican
+  variant_name: null
+- slug: czech
+  code: cs
+  base_code: cs
+  name: Czech
+  variant_name: null
+- slug: welsh
+  code: cy
+  base_code: cy
+  name: Welsh
+  variant_name: null
+- slug: danish
+  code: da
+  base_code: da
+  name: Danish
+  variant_name: null
+- slug: german
+  code: de
+  base_code: de
+  name: German
+  variant_name: null
+- slug: german
+  code: de-at
+  base_code: de
+  name: German
+  variant_name: null
+- slug: german
+  code: de-ch
+  base_code: de
+  name: German
+  variant_name: null
+- slug: german
+  code: de-de
+  base_code: de
+  name: German
+  variant_name: null
```

Comment:
Not sure if we can implement anything here, but perhaps there is a better way to display locales support here?
We usually follow this structure when the ISO language code used by the API is atypical, but doesn't make much sense for locales, which are not atypical.
What do you think?

![Screenshot 2023-09-17 at 15 14 35](https://github.com/machinetranslate/machinetranslate.org/assets/33723604/24c2c7db-8ee0-426c-bfdb-c4f235d070f0)

Reply:
Hi @cefoo

It's great to hear that everything worked properly!

I will definitely look into the points that you have mentioned. Thank you very much for pointing them out!